xbmc/music/Album.cpp

```diff
@@ -484,11 +487,21 @@ bool CAlbum::Load(const TiXmlElement *album, bool append, bool prioritise)
   XMLUtils::GetBoolean(album, "boxset", bBoxedSet);
 
   XMLUtils::GetString(album,"review",strReview);
-  XMLUtils::GetString(album,"releasedate",m_strDateOfRelease);
   XMLUtils::GetString(album,"label",strLabel);
   XMLUtils::GetString(album,"type",strType);
 
-  XMLUtils::GetInt(album,"year",iYear);
+  XMLUtils::GetString(album, "releasedate", strReleaseDate);
+  StringUtils::Trim(strReleaseDate);  // @todo: validate ISO8601 format
+  // Support old style <year></year> for backwards compatibility
+  if (strReleaseDate.empty())
+  {
+    int year;
+    if (XMLUtils::GetInt(album, "year", year));
+    if (year > 0)
+      strReleaseDate = StringUtils::Format("%04i", year);
```

Comment:
Misplaced `;` don't really need if at all
This is the compile warning issue on some platforms hence jenkins fail

Reply:
Sorted.  Thanks.